patches/chrome-browser-ui-views-location_bar-icon_label_bubble_view.cc.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/chrome/browser/ui/views/location_bar/icon_label_bubble_view.cc b/chrome/browser/ui/views/location_bar/icon_label_bubble_view.cc
+index f08cd04d815b30bfc089d5a94a4928c3d627fffd..64ddfa54c277f115b03290e64b605d33169f8ce7 100644
+--- a/chrome/browser/ui/views/location_bar/icon_label_bubble_view.cc
++++ b/chrome/browser/ui/views/location_bar/icon_label_bubble_view.cc
+@@ -567,7 +567,7 @@ SkPath IconLabelBubbleView::GetHighlightPath() const {
+ 
+   const float corner_radius = highlight_bounds.height() / 2.f;
+   const SkRect rect = RectToSkRect(highlight_bounds);
+-
```

Comment:
This should only contain an additional line, not a removal of the blank line also which it looks like it has.